churchland_pipeline/acquisition.py

```diff
@@ -0,0 +1,147 @@
+import datajoint as dj
+from . import lab, ephys, reference
+
+schema = dj.schema('churchland_acquisition')
+
+# -------------------------------------------------------------------------------------------------------------------------------
+# LEVEL 0
+# -------------------------------------------------------------------------------------------------------------------------------
+
+@schema
+class Session(dj.Manual):
+    definition = """
+    # Recording session
+    session_date: date # session date
+    -> lab.Monkey
+    ---
+    -> lab.User
+    -> [nullable] lab.User.proj(user2 = 'user') # secondary experimenter
+    -> [nullable] lab.User.proj(user3 = 'user') # tertiary experimenter
+    """
+    
+    class Notes(dj.Part):
+        definition = """
+        # Session notes
+        -> master
+        ---
+        session_notes: varchar(8192)
+        """
+        
+    class SaveTag(dj.Part):
+        definition = """
+        # Save tags and associated notes
+        -> master
+        ---
+        save_tag: tinyint unsigned # save tag
+        save_tag_notes: varchar(2048) # notes for the save tag
+        """
+        
+# -------------------------------------------------------------------------------------------------------------------------------
+# LEVEL 1
+# ------------------------------------------------------------------------------------------------------------------------------- 
+
+@schema
+class EphysRecording(dj.Imported):
+    definition = """
+    -> Session
+    ---
+    ephys_file_path: varchar(512) # file path
+    ephys_file_name: varchar(256) # file name
+    """
+    
+    class Meta(dj.Part):
+        definition = """
+        # Meta parameters for Ephys recording
+        -> master
+        ---
+        ephys_sample_rate : smallint unsigned # sample rate [Hz]
+        ephys_time_stamp : double # clock start time [sec]
+        ephys_data_samples : int unsigned # recording duration [samples]
+        ephys_data_duration : double # recording duration [sec]
+        ephys_channel_count : smallint unsigned # number of channels on the recording file
+        """
+    
+@schema
+class ProblematicSession(dj.Manual):
+    definition = """
+    # Problematic sessions
+    -> Session
+    ---
+    problem_reason = '' : varchar(256)
```

Comment:
This is nit picky, but `varchar(255)` requires 1 byte for storing the string length whereas  `varchar(256)` requires two bytes, same as `varchar(65535)`.  I recommend using `varchar(255)` as a general short string.